README.md

```diff
@@ -1,34 +1,51 @@
-This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
+# A Next.js OAuth Agent for SPAs
 
-## Getting Started
+[![Quality](https://img.shields.io/badge/quality-test-yellow)](https://curity.io/resources/code-examples/status/)
+[![Availability](https://img.shields.io/badge/availability-source-blue)](https://curity.io/resources/code-examples/status/)
 
-First, run the development server:
+## Overview
 
-```bash
-npm run dev
-# or
-yarn dev
-```
+The OAuth Agent acts as a modern `Back End for Front End (BFF)` for Single Page Applications.\
+This implementation demonstrates the standard pattern for SPAs:
 
-Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
+- Strongest browser security with only `SameSite=strict` cookies
```

Comment:
I advice against the use of superlative as it requires a comparison and discussion (strongest compared to what).
Also, the cookie returned is assigned the attribute `HTTP only`.
Suggest edit:
- Strong browser security with `HTTP only` and `SameSite=strict` cookies